magefiles/installation/install.go

```diff
@@ -98,6 +113,29 @@ func NewAppStudioInstallController() (*InstallAppStudio, error) {
 	}, nil
 }
 
+func NewAppStudioInstallControllerDefault() (*InstallAppStudio, error) {
+	cwd, _ := os.Getwd()
+	k8sClient, err := kubeCl.NewAdminKubernetesClient()
+
+	if err != nil {
+		return nil, err
+	}
+
+	return &InstallAppStudio{
+		KubernetesClient:                 k8sClient,
+		TmpDirectory:                     DEFAULT_TMP_DIR,
+		InfraDeploymentsCloneDir:         fmt.Sprintf("%s/%s/infra-deployments-upgrade", cwd, DEFAULT_TMP_DIR),
+		InfraDeploymentsBranch:           DEFAULT_INFRA_DEPLOYMENTS_BRANCH,
+		InfraDeploymentsOrganizationName: DEFAULT_INFRA_DEPLOYMENTS_GH_ORG,
+		LocalForkName:                    DEFAULT_LOCAL_FORK_NAME,
+		LocalGithubForkOrganization:      utils.GetEnv("MY_GITHUB_ORG", DEFAULT_LOCAL_FORK_ORGANIZATION),
+		E2EApplicationsNamespace:         utils.GetEnv("E2E_APPLICATIONS_NAMESPACE", DEFAULT_E2E_APPLICATIONS_NAMEPSPACE),
```

Comment:
Is this variable needed here? (It's not being used in `NewAppStudioInstallController()`)

Reply:
Thank you - I have removed the *Default function completely.